scripts/create_addon

```diff
@@ -56,7 +56,7 @@ pack_addon() {
     fi
     cd $ADDON_BUILD
     echo "*** compressing Addon $PKG_ADDON_ID ... ***"
-    zip -rq $PKG_ADDON_ID-$ADDONVER.zip $PKG_ADDON_ID;
+    $TOOLCHAIN/bin/7za a -l -mx9 -bsp0 -bso0 $PKG_ADDON_ID-$ADDONVER.zip $PKG_ADDON_ID;
```

Comment:
Symbolic links save space indeed but Kodi can't handle them. It unpacks symlinks as regular files.